architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,81 +341,149 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
-    - created: 2016-01-29T13:40:11Z
+- created: 2017-09-02T10:15:09Z
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
+      generation: 2
+      image: sha256:909de62d1f609a717ec433cc25ca5cf00941545c83a01fb31527771e1fab3fc5 <3>
+    - created: 2017-09-29T13:40:11Z
+      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:909de62d1f609a717ec433cc25ca5cf00941545c83a01fb31527771e1fab3fc5
       generation: 1
       image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+    tag: latest <4>
 ----
+
+<1> The name of the image stream.
+<2> The image source repository.
+<3> The internal registry service IP address, which is the target for pushes to the image stream. This value is set automatically.
+<3> The SHA identifier that this image stream tag currently references. Resources that reference this image stream tag use this identifier.
```

Comment:
Couldn't find the 'The SHA identifier that this image stream tag currently references' ?

Reply:
@zhouying7780 Thank you. I fixed the call-out numbering. The SHA identifier is <4>:
`image: sha256:909de62d1f609a717ec433cc25ca5cf00941545c83a01fb31527771e1fab3fc5 <4>`